format/src/ber_c/system.py

```
@@ -1324,15 +1318,25 @@ def _get_packed_value(self, packed_base_value):
         return packed_value
 
     def _get_pack_method(self, type_number):
+        # raises an error in case the type number is not found
+        if not type_number in self.unpack_methods_map:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)